Product/Production/Deploy/pom.xml

```diff
@@ -11,6 +11,14 @@
   <packaging>pom</packaging>
   <modelVersion>4.0.0</modelVersion>
   
+    <properties>
+        <db.host>localhost</db.host>
+        <db.password>nhincpass</db.password>
```

Comment:
Is it better to keep the username password in different settings file rather than in the pom?

Reply:
This is already defined in the SoapUI_Test pom. So to be consistent I had to place it here.